lib/util/mail.php

```diff
@@ -104,6 +104,18 @@ function mail_utf8($to, $from_user, $from_email, $subject = '(No subject)', $mes
         return mail_ses($to, $subject, $message);
     }
 
+    if (getenv('MAIL_DRIVER') === 'mailcatcher') {
+        $transport = (new Swift_SmtpTransport('mailcatcher', 1025));
```

Comment:
I'm not sure the outer parentheses for this initialization are really necessary, but I won't hold the review back over it.

Reply:
Can be removed, agreed.